2021-03-05 17:12:57.297909: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer.so.6'; dlerror: libnvinfer.so.6: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/Cluster-Apps/graphviz/2.40.1/lib:/usr/local/Cluster-Apps/cudnn/7.6_cuda-10.1/lib64:/usr/local/software/cuda/10.1/lib64:/usr/local/software/cuda/10.1/lib:/usr/local/software/cuda/10.1/nvvm/lib64:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/tbb/lib/intel64_lin/gcc4.7:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/compiler/lib/intel64_lin:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/mkl/lib/intel64_lin:/usr/local/software/master/python/3.6/lib64:/usr/local/software/master/python/3.6/lib:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-5.4.0/openmpi-1.10.7-jdc7f4fjdq5roxhadufd6h66xkwuytss/lib:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-4.8.5/gcc-5.4.0-fis24ggupugiobii56fesif2y3qulpdr/lib64:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-4.8.5/gcc-5.4.0-fis24ggupugiobii56fesif2y3qulpdr/lib:/usr/local/software/global/lib:/usr/local/Cluster-Apps/vgl/2.5.1/64/lib:/usr/local/software/slurm/current/lib
2021-03-05 17:12:57.301586: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libnvinfer_plugin.so.6'; dlerror: libnvinfer_plugin.so.6: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/Cluster-Apps/graphviz/2.40.1/lib:/usr/local/Cluster-Apps/cudnn/7.6_cuda-10.1/lib64:/usr/local/software/cuda/10.1/lib64:/usr/local/software/cuda/10.1/lib:/usr/local/software/cuda/10.1/nvvm/lib64:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/tbb/lib/intel64_lin/gcc4.7:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/compiler/lib/intel64_lin:/usr/local/software/archive/linux-scientific7-x86_64/gcc-9/intel-mkl-2020.1.217-daoabwrh3hmki2sfkvbm34nozs6ehxy5/compilers_and_libraries_2020.1.217/linux/mkl/lib/intel64_lin:/usr/local/software/master/python/3.6/lib64:/usr/local/software/master/python/3.6/lib:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-5.4.0/openmpi-1.10.7-jdc7f4fjdq5roxhadufd6h66xkwuytss/lib:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-4.8.5/gcc-5.4.0-fis24ggupugiobii56fesif2y3qulpdr/lib64:/usr/local/software/spack/spack-0.11.2/opt/spack/linux-rhel7-x86_64/gcc-4.8.5/gcc-5.4.0-fis24ggupugiobii56fesif2y3qulpdr/lib:/usr/local/software/global/lib:/usr/local/Cluster-Apps/vgl/2.5.1/64/lib:/usr/local/software/slurm/current/lib
2021-03-05 17:12:57.301605: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:30] Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
Returning A...
  (0, 1)	1.0
  (0, 2)	1.0
  (0, 48)	1.0
  (0, 49)	1.0
  (0, 50)	1.0
  (0, 96)	1.0
  (0, 97)	1.0
  (0, 98)	1.0
  (1, 0)	1.0
  (1, 2)	1.0
  (1, 3)	1.0
  (1, 48)	1.0
  (1, 49)	1.0
  (1, 50)	1.0
  (1, 96)	1.0
  (1, 97)	1.0
  (2, 0)	1.0
  (2, 1)	1.0
  (2, 3)	1.0
  (2, 4)	1.0
  (2, 48)	1.0
  (2, 49)	1.0
  (2, 50)	1.0
  (2, 51)	1.0
  (2, 98)	1.0
  :	:
  (2300, 2302)	1.0
  (2301, 2205)	1.0
  (2301, 2252)	1.0
  (2301, 2253)	1.0
  (2301, 2254)	1.0
  (2301, 2299)	1.0
  (2301, 2300)	1.0
  (2301, 2302)	1.0
  (2301, 2303)	1.0
  (2302, 2206)	1.0
  (2302, 2252)	1.0
  (2302, 2253)	1.0
  (2302, 2254)	1.0
  (2302, 2255)	1.0
  (2302, 2300)	1.0
  (2302, 2301)	1.0
  (2302, 2303)	1.0
  (2303, 2205)	1.0
  (2303, 2206)	1.0
  (2303, 2207)	1.0
  (2303, 2253)	1.0
  (2303, 2254)	1.0
  (2303, 2255)	1.0
  (2303, 2301)	1.0
  (2303, 2302)	1.0
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
Traceback (most recent call last):
  File "/home/ir-jaco1/layers3.py", line 349, in <module>
    X_train, y_train, X_val, y_val, X_test, y_test, A, pos = mnist_regular_graph(k=8)  # k=number of next neighbors
  File "/home/ir-jaco1/layers3.py", line 334, in mnist_regular_graph
    X_train, y_train, X_val, y_val, X_test, y_test, A = load_data(k)
  File "/home/ir-jaco1/layers3.py", line 180, in load_data
    x,y=generate_training_data(filelist)
  File "/home/ir-jaco1/layers3.py", line 138, in generate_training_data
    inputdata = h5py.File(file, 'r')
  File "/home/ir-jaco1/t2/lib/python3.6/site-packages/h5py/_hl/files.py", line 408, in __init__
    swmr=swmr)
  File "/home/ir-jaco1/t2/lib/python3.6/site-packages/h5py/_hl/files.py", line 173, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (truncated file: eof = 63647744, sblock->base_addr = 0, stored_eof = 124441653)
Returning A...
  (0, 1)	1.0
  (0, 2)	1.0
  (0, 48)	1.0
  (0, 49)	1.0
  (0, 50)	1.0
  (0, 96)	1.0
  (0, 97)	1.0
  (0, 98)	1.0
  (1, 0)	1.0
  (1, 2)	1.0
  (1, 3)	1.0
  (1, 48)	1.0
  (1, 49)	1.0
  (1, 50)	1.0
  (1, 96)	1.0
  (1, 97)	1.0
  (2, 0)	1.0
  (2, 1)	1.0
  (2, 3)	1.0
  (2, 4)	1.0
  (2, 48)	1.0
  (2, 49)	1.0
  (2, 50)	1.0
  (2, 51)	1.0
  (2, 98)	1.0
  :	:
  (2300, 2302)	1.0
  (2301, 2205)	1.0
  (2301, 2252)	1.0
  (2301, 2253)	1.0
  (2301, 2254)	1.0
  (2301, 2299)	1.0
  (2301, 2300)	1.0
  (2301, 2302)	1.0
  (2301, 2303)	1.0
  (2302, 2206)	1.0
  (2302, 2252)	1.0
  (2302, 2253)	1.0
  (2302, 2254)	1.0
  (2302, 2255)	1.0
  (2302, 2300)	1.0
  (2302, 2301)	1.0
  (2302, 2303)	1.0
  (2303, 2205)	1.0
  (2303, 2206)	1.0
  (2303, 2207)	1.0
  (2303, 2253)	1.0
  (2303, 2254)	1.0
  (2303, 2255)	1.0
  (2303, 2301)	1.0
  (2303, 2302)	1.0
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
SHAPE IS: 2000
Traceback (most recent call last):
  File "/home/ir-jaco1/layers3.py", line 349, in <module>
    X_train, y_train, X_val, y_val, X_test, y_test, A, pos = mnist_regular_graph(k=8)  # k=number of next neighbors
  File "/home/ir-jaco1/layers3.py", line 334, in mnist_regular_graph
    X_train, y_train, X_val, y_val, X_test, y_test, A = load_data(k)
  File "/home/ir-jaco1/layers3.py", line 180, in load_data
    x,y=generate_training_data(filelist)
  File "/home/ir-jaco1/layers3.py", line 138, in generate_training_data
    inputdata = h5py.File(file, 'r')
  File "/home/ir-jaco1/t2/lib/python3.6/site-packages/h5py/_hl/files.py", line 408, in __init__
    swmr=swmr)
  File "/home/ir-jaco1/t2/lib/python3.6/site-packages/h5py/_hl/files.py", line 173, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (truncated file: eof = 63647744, sblock->base_addr = 0, stored_eof = 124441653)
